### PR TITLE
Improve test for consumer with inactivity threshold

### PR DESCRIPTION
**Pull request by Jarema**
This test adds check if consumer with inactivity threshold is properly removed on intertest based stream when reaching the threshhold.

Signed-off-by: Tomasz Pietrek <tomasz@nats.io>

/cc @nats-io/core

### CODE REVIEW

File: server/jetstream_cluster_3_test.go [hunk 1]
```diff
@@ -2767,6 +2765,17 @@ func TestJetStreamClusterInterestPolicyEphemeral(t *testing.T) {
...
+			// Stop the subscription and remove the interest.
+			err = sub.Unsubscribe()
+			require_NoError(t, err)
+
+			// Now wait for interest inactivity threshold to kick in.
+			time.Sleep(3 * inactiveThreshold / 2)
+
+			// Check if the consumer has been removed.
+			_, err = js.ConsumerInfo(test.stream, name)
+			require_Error(t, err)
```
Comment (derekcollison) [inline server/jetstream_cluster_3_test.go:2778]: Suggest make it explicit.

require_Error(t, err, nats.ConsumerNotFound)

**Review by derekcollison — CHANGES_REQUESTED** [review]
Just one small nit.

File: server/jetstream_cluster_3_test.go (hunk 1 above)
Comment (Jarema) [reply server/jetstream_cluster_3_test.go]: ah, good idea. fixed.

**Review by derekcollison — APPROVED** [review]
LGTM!

**Pull request merged**